package/python-kiwi-spec-template

```diff
@@ -79,13 +86,20 @@ Source:         %{name}.tar.gz
 Source1:        %{name}-boot-packages
 Source2:        %{name}-rpmlintrc
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
-BuildRequires:  python3-devel
-BuildRequires:  python3-setuptools
 BuildRequires:  python-devel
 BuildRequires:  python-setuptools
+%if 0%{?fedora_version} || 0%{?suse_version}
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+%endif
+%if 0%{?suse_version}
 BuildRequires:  fdupes
-BuildRequires:  update-alternatives
 BuildRequires:  shadow
+BuildRequires:  update-alternatives
+%endif
+%if 0%{?fedora_version} || 0%{?rhel_version} || 0%{?centos_version}
```

Comment:
Again, please use `%{fedora}`.

In addition, `%rhel_version` and `%centos_version` can be combined into a single `%{rhel}` check.